src/node_file.cc

```diff
@@ -1853,6 +1880,21 @@ static void Open(const FunctionCallbackInfo<Value>& args) {
   CHECK(args[2]->IsInt32());
   const int mode = args[2].As<Int32>()->Value();
 
+  // Open can be called either in write or read
+  if (flags == O_RDWR) {
+    // TODO(rafaelgss): it can be optimized to void n*m
```

Comment:
to *avoid n*m

Reply:
Ah, okay! But what's n and m here?

Reply:
It's because I'm calling THROW_IF_... twice. One for write access, and another for read access. It can be a single call with both parameters, so I won't need to iterate to the array twice. Actually, the correct sentence would be: "avoid n*2"

Reply:
Oh, it's regarding complexity. Maybe the Big O notation would be expressive? (I originally assumed that n and m were variables which I wasn't able to find lol)
```suggestion
    // TODO(rafaelgss): it can be optimized to avoid O(2 * n)
```